docs/src/components/home/ToolpadShowcase.tsx

```diff
@@ -0,0 +1,257 @@
+})
+  `;
+
+const tabsCustomStyles = {
+  minHeight: '26px',
+  p: 0,
+  borderBottom: '1px solid',
+  borderColor: 'divider',
+  '& .MuiTabs-flexContainer': {
+    p: 1,
+    gap: '6px',
+  },
+  '& .MuiTab-root': {
+    minHeight: '26px',
+    minWidth: 'fit-content',
+    p: '6px',
+    borderRadius: '6px',
+    fontSize: '.75rem',
+    fontWeight: 'medium',
+    lineHeight: 1,
+    '&:hover': {
+      backgroundColor: 'primaryDark.700',
+    },
+  },
+  '& .MuiTabs-indicator': {
+    height: '1px',
+    opacity: '60%',
+  },
+  '& .Mui-selected': {
+    color: 'primary.300',
+  },
+};
+
+interface ImageProps {
+  alt: string;
+  index: number;
+  src: string;
+  value: number;
+}
+
+function Image({ alt, index, src, value }: ImageProps) {
+  return (
+    <Box
+      component="img"
+      hidden={value !== index}
+      src={src}
+      alt={alt}
+      loading="lazy"
+      height={260}
+      sx={{
+        width: { xs: 'auto', sm: '100%' },
+        objectFit: 'cover',
+        objectPosition: 'right',
+      }}
+    />
+  );
+}
+
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: number;
+  value: number;
+}
+
+function CustomTabPanel({ children, value, index, ...other }: TabPanelProps) {
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`toolpad-showcase-tabpanel-${index}`}
+      aria-labelledby={`toolpad-showcase-${index}`}
+      {...other}
+    >
+      {value === index && <React.Fragment>{children}</React.Fragment>}
+    </div>
+  );
+}
+
+function a11yProps(index: number) {
+  return {
+    id: `toolpad-showcase-tab-${index}`,
+    'aria-controls': `toolpad-showcase-tab-${index}`,
+  };
+}
+
+const tabsCodeInfo = [
+  {
+    code: tabOneCode,
+    language: 'tsx',
+    description:
+      "Write serverless functions that access your project's code, and let Toolpad handle linking your data with the UI.",
+  },
+  {
+    code: tabTwoCode,
+    language: 'yaml',
+    description:
+      'Store your app configuration locally in yaml files. Changes in Toolpad are automatically synced to the files, and vice-versa.',
+  },
+  {
+    code: tabThreeCode,
+    language: 'tsx',
+    description: 'Compose your UI with drag and drop using your own React components.',
+  },
+];
+
+export default function ToolpadShowcase() {
+  const [value, setValue] = React.useState(0);
+
+  const handleChange = (event: React.SyntheticEvent, newValue: number) => {
+    setValue(newValue);
+  };
+
+  return (
+    <ShowcaseContainer
+      noPadding
+      preview={
+        <Box sx={{ width: '100%', display: 'flex', flexDirection: 'column' }}>
+          <Tabs
+            value={value}
+            onChange={handleChange}
+            aria-label="Toolpad showcase"
+            sx={tabsCustomStyles}
+          >
+            <Tab label="Serverless functions" {...a11yProps(0)} />
+            <Tab label="yaml files" {...a11yProps(1)} />
+            <Tab label="React components" {...a11yProps(2)} />
```

Comment:
```suggestion
            <Tab label="Customize" {...a11yProps(2)} />
```

These are just suggestions. I'd also consider reordering

* Local first
* Serverless functions
* Customize